### PR TITLE
PEER20-349 UI: Fix errors and warnings in console

### PR DESCRIPTION
**Pull request by AweAwe95**

### CODE REVIEW

File: app/containers/Questions/Content/Body/IPFSInformation.js [hunk 1]
```diff
@@ -73,13 +73,19 @@ const IPFSInformation = ({ locale, ipfsHash, histories }) => {
       {formattedData?.length > 0 && (
         <table className="table mt-1 mb-0">
           <thead>
-            {Object.values(columns).map(column => <th>{column}</th>)}
```
Comment (AweAwe95) [inline app/containers/Questions/Content/Body/IPFSInformation.js:76]: I'm not sure if these are ideal values ​​for keys, but in my opinion this is the only correct solution

File: app/containers/Questions/Content/Body/IPFSInformation.js [hunk 2]
```diff
@@ -73,13 +73,19 @@ const IPFSInformation = ({ locale, ipfsHash, histories }) => {
...
           </thead>
 
           <tbody>
             {formattedData.map(item => (
-              <tr>
-                {Object.keys(columns).map(column => <td>{item[column]}</td>)}
+              <tr key={item.transactionHash.props.children}>
+                {Object.keys(columns).map(column => (
+                  <td key={column}>{item[column]}</td>
+                ))}
```
Comment (AweAwe95) [inline app/containers/Questions/Content/Body/IPFSInformation.js:88]: I'm not sure if these are ideal values ​​for keys, but in my opinion this is the only correct solution

**Comment by sergeyi23** [comment]
@Lampardks , please review this PR.

File: app/containers/Questions/Content/Body/IPFSInformation.js (hunk 2 above)
Comment (Lampardks) [reply app/containers/Questions/Content/Body/IPFSInformation.js:88]: column unique value for each key?

Comment (AweAwe95) [reply app/containers/Questions/Content/Body/IPFSInformation.js:88]: done

**Review by Lampardks — APPROVED** [review]

**Pull request merged**